### ISSUE TITLE
Disable "Notify when sponsor segment is skipped" when u disable "Enable SponsorBlock"

### ISSUE THREAD

**Issue by efb4f5ff-1298-471a-8973-3d47447115dc** [FreeTubeApp/FreeTube#1468]
**Is your feature request related to a usage problem (not a bug)? Please describe.**
"Notify when sponsor segment is skipped" does nothing when Sponserblock is disabled. 

**Describe the solution you'd like to see implemented**
Disable the setting "Notify when sponsor segment is skipped" when u click on the toggle for disabling Sponserblock.

**Describe alternatives you've considered**
None.

**Screenshots**
None.

**Additional context**
None.

**Comment by jasonhenriquez** [comment]
> When Expand Side Bar by Default is enabled and u enable Hide Side Bar Labels it needs to disable Expand Side Bar by Default and also the other way around.

Can you explain why this behavior is needed? These features are not mutually exclusive. The normal expanded sidebar can still be accessed even with "Hide Side Bar Labels" enabled.

**Comment by jasonhenriquez** [comment]
> Because hide label gets overwritten by expand sidebar.

This is not true. This setting expands the side bar on click, on startup, & on reset, but the side bar can be compressed at any time with the hamburger button.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Ah i see, ur right!
